packages/react/src/components/Forms/Select/mono.spec.tsx

```diff
@@ -139,6 +141,141 @@ describe("<Select/>", () => {
+                  value: "paris",
+                },
+                {
+                  label: "Panama",
+                  value: "panama",
+                },
+                {
+                  label: "London",
+                  value: "london",
+                },
+                {
+                  label: "New York",
+                  value: "new_york",
+                },
+                {
+                  label: "Tokyo",
+                  value: "tokyo",
+                },
+              ]}
+              searchable={true}
+              value={value}
+              onChange={(e) => setValue(e.target.value as any)}
+            />
+            <Input name="Something else" />
+          </CunninghamProvider>
+        );
+      };
+      render(<Wrapper />);
+
+      const user = userEvent.setup();
+      const input = screen.getByRole("combobox", {
+        name: "City",
+      });
+      screen.getByText("Value = |");
+      // It returns the input.
+      expect(input.tagName).toEqual("INPUT");
+
+      const menu: HTMLDivElement = screen.getByRole("listbox", {
+        name: "City",
+      });
+
+      expectMenuToBeClosed(menu);
+
+      // Click on the input.
+      await user.click(input);
+      expectMenuToBeOpen(menu);
+      expectOptions(["Paris", "Panama", "London", "New York", "Tokyo"]);
+
+      // Verify that filtering works.
+      await user.type(input, "Pa");
+      expectMenuToBeOpen(menu);
+      expectOptions(["Paris", "Panama"]);
+
+      await user.type(input, "rr");
+      expectNoOptions();
+
+      expect(input).toHaveValue("Parr");
+
+      // This is a way to blur the combobox.
+      await user.click(screen.getByRole("textbox"));
+
+      expect(input).toHaveValue("");
+      screen.getByText("Value = |");
+    });
+    it("clears the added text to the existing value input on blur if no other item is selected", async () => {
+      const Wrapper = () => {
+        const [value, setValue] = useState<string | number | undefined>(
+          "london",
+        );
+        return (
+          <CunninghamProvider>
+            <div>Value = {value}|</div>
+            <Select
+              label="City"
+              options={[
+                {
+                  label: "Paris",
+                  value: "paris",
+                },
+                {
+                  label: "Panama",
+                  value: "panama",
+                },
+                {
+                  label: "London",
+                  value: "london",
+                },
+                {
+                  label: "New York",
+                  value: "new_york",
+                },
+                {
+                  label: "Tokyo",
+                  value: "tokyo",
+                },
+              ]}
+              searchable={true}
+              value={value}
+              onChange={(e) => setValue(e.target.value as any)}
+            />
+            <Input name="Something else" />
```

Comment:
FMI, what is the purpose of `<Input`?

Reply:
You can see here that this is only to trigger the onBlur event :)

https://github.com/openfun/cunningham/blob/f8de58cbf46317a845ab9d590aa173d22f3b9da9/packages/react/src/components/Forms/Select/mono.spec.tsx#L273